data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,851 @@
+	description `	Guile Pulse Lasers have noticeable drawbacks as they quickly consume energy and generate notable heat per shot. Unlike a beam-type weapon, they are incapable of consistent output, and have a fairly short range.`
+
+
+outfit "Excavator"
+	category "Guns"
+	cost 500000
+	"thumbnail" "outfit/excavator"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
+	"gun ports" -1
+	"drill port" -1
+	"drill lock" 1
+	weapon
+		sound "drill"
+		"hit effect" "drilling"
+		"inaccuracy" 90
+		"velocity" 18
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 4
+		"firing heat" 100
+		"shield damage" 8
+		"hull damage" 14
+		"minable damage" 104
+		"hit force" -4
+	description `Excavator Drills are large drills designed by the Gegno that rip materials out of asteroids in an efficient manner. These types of drill systems are only found on ships specifically designed around them as they require substantial infrastructure to be functional.`
+
+effect "drilling"
+	sprite "effect/smoke"
+		"no repeat"
+	sound "explosion tiny"
+	"lifetime" 20
+	"random angle" 360
+	"random frame rate" 5
+	"random spin" 2
+	"random velocity" 0.4
+	"velocity scale" -0.01
+
+
+outfit "Savagery Bludgeon"
+	category "Guns"
+	cost 10000
+	thumbnail "outfit/unknown"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"gun ports" -1
+	"unplunderable" 1
+	weapon
+		"hit effect" "bludgeon impact"
+		"inaccuracy" 360
+		"velocity" 30
+		"lifetime" 1
+		"reload" 1
+		"shield damage" 8
+		"hull damage" 40
+		"hit force" 140
+	description `A Savagery Bludgeon is a mace-like weapon found jammed into the Gegno Vi's Dunite duelist ships. They are used as melee weapons for close-quarter rams and jousts during duels. However, they barely have any use as a weapon outside of this as they merely scratch larger ship's hulls and barely put a dent into anything shielded.`
+	
+effect "bludgeon impact"
+	sound "bludgeon impact"
+
+
+# Special Outfits
+
+outfit "Gegno Civilian License"
+	category "Licenses"
+	thumbnail "outfit/unknown"
+	description "You have been accepted as a titular Gegno citizen that provides benefits for the Gegno race, and have been awarded access to their civilian technology."
+
+outfit "Vi Lord License"
+	category "Licenses"
+	thumbnail "outfit/unknown"
+	description "A majority of the Vi's military hierarchy recognizes you as a respectable foe and invaluable asset for their ambitions. The responsibility of maintaining the largest of Vi vessels is now within your grasp."
+	
+outfit "Vi Centurion License"
+	category "Licenses"
+	thumbnail "outfit/unknown"
+	description "Amongst the Vi's Evocati you have proven yourself as a powerful leader in combat and a successful user of judgment. Access to a portion of the Vi's military arsenal awaits your interests."
```

Comment:
This has an... Interesting rythmn to it. Is judgment a thing in the story? or is this just saying that the player has good judgement? The way it reads makes it sound like judgement is a weapon or piece of equipment that the player has succesfully used.

I'm not suggesting anything because this could just be the way the Vi view judgement, and thus is culturally appropriate.

Reply:
Centurion is a step above Evocati, where Evocati is a sort of "foot soldier". Vi rankings are based on Roman army titles. So, in this case, a Centurion has been helpful to the Vi and had access to some of their tech. I very much would like the player, if they choose to actually participate in the war, be restricted and given access to Gegno tech during that period.

From Wikipidea:

An evocatus (plural evocati) was a soldier in the Ancient [Roman army] who had served out his time and obtained an honorable discharge [honesta missio]) but had voluntarily enlisted again at the invitation of the [consul] or other commander.

A centurion was a position in the [Roman army], nominally the commander of a [century], a [military unit] of around 80 [legionaries].

[Vi Lord] I did myself though. Also, the Scin's "licenses" are based on real-life field names.

Reply:
Commenting on the Mace (these are so close Git combined this suggestion lol), it is more of a protrusion yes. Derpy as stated above suggested a reload longer for it. I don't think the damage really needs to be upped though, at least not terribly much more- it's done as a beam right now. but a large knockback force balances it as the ship will be slung backwards after a hit.